unit-tests/post-processing/test-post-processing-filters.py

```diff
@@ -0,0 +1,469 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#temporary fix to prevent the test from running on Win_SH_Py_DDS_CI
+#test:donotrun:dds
+
+import pyrealsense2 as rs, tempfile, os, random, csv
+from rspy import test, repo
+################################################################################################
+frame_metadata_count = 43
+
+ppf_test_cases = [
+    # All the tests below include depth-disparity domain transformation
+    # Downsample scales 2 and 3 are tested only. scales 4-7 are differ in impementation from the reference code:
+    # In Librealsense for all scales [2-8] the filter is the mean of depth.
+    # I the reference code for [2-3] the filter uses mean of depth, and for 4-7 is switches to man of disparities doe to implementation constrains
+    ("1551257764229", "D435_DS(2)"),
+    ("1551257812956", "D435_DS(3)"),
+    # Downsample + Hole-Filling modes 0/1/2
+    ( "1551257880762","D435_DS(2)_HoleFill(0)" ),
+    ( "1551257882796","D435_DS(2)_HoleFill(1)" ),
+    ( "1551257884097","D435_DS(2)_HoleFill(2)" ),
+    # Downsample + Spatial Filter parameters
+    ( "1551257987255",  "D435_DS(2)+Spat(A:0.85/D:32/I:3)" ),
+    ( "1551259481873",  "D435_DS(2)+Spat(A:0.5/D:15/I:2)" ),
+    #Downsample + Temporal Filter
+    ( "1551261946511",  "D435_DS(2)+Temp(A:0.25/D:15/P:0)" ),
+    ( "1551262153516",  "D435_DS(2)+Temp(A:0.45/D:25/P:1)" ),
+    ( "1551262256875",  "D435_DS(2)+Temp(A:0.5/D:30/P:4)" ),
+    ( "1551262841203",  "D435_DS(2)+Temp(A:0.5/D:30/P:6)" ),
+    ( "1551262772964",  "D435_DS(2)+Temp(A:0.5/D:30/P:8)" ),
+    # Downsample + Spatial + Temporal
+    ( "1551262971309",  "D435_DS(2)_Spat(A:0.7/D:25/I:2)_Temp(A:0.6/D:15/P:6)" ),
+    # Downsample + Spatial + Temporal (+ Hole-Filling)
+    ( "1551263177558",  "D435_DS(2)_Spat(A:0.7/D:25/I:2)_Temp(A:0.6/D:15/P:6))_HoleFill(1)" ),
+]
+
+metadata_attributes = {
+    'res_x':        "Resolution_x",
+    'res_y':        "Resolution_y",
+    'focal_length':  "Focal Length",
+    'depth_units':   "Depth Units",
+    'stereo_baseline':"Stereo Baseline",
+    'downscale':    "Scale",
+    'spat_filter':   "Spatial Filter Params:",
+    'spat_alpha':   "SpatialAlpha",
+    'spat_delta':   "SpatialDelta",
+    'spat_iter':    "SpatialIterations",
+    'temp_filter':  "Temporal Filter Params:",
+    'temp_alpha':   "TemporalAlpha",
+    'temp_delta':   "TemporalDelta",
+    'temp_persist': "TemporalPersistency",
+    'holes_filter': "Holes Filling Mode:",
+    'holes_fill':   "HolesFilling",
+    'frames_sequence_size': "Frames sequence length"
+}
+class post_proccesing_filters:
+    def __init__(self):
+        self.depth_to_disparity = rs.disparity_transform(True)
+        self.disparity_to_depth = rs.disparity_transform(False)
+        self.dec_pb = False
```

Comment:
Rename `xxx_pb` members to `use_xxx`. This aren't the processing blocks, just configuration flags.

Reply:
Done